src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -97,13 +97,17 @@ def load_arguments(self, _):
 
     with self.argument_context('backup vault create') as c:
         c.argument('tags', arg_type=tags_type)
+        c.argument('classic_alerts', arg_type=get_enum_type(['Enable', 'Disable']), help='Use this property to specify whether backup alerts from the classic solution should be received.')
+        c.argument('azure_monitor_alerts_for_job_failures', arg_type=get_enum_type(['Enable', 'Disable']), help='Use this property to specify whether built-in Azure Monitor alerts should be received for every job failure.')
```

Comment:
This parameter name is too long, which will make it inconvenient for the user to input. Could you change the parameter name to be shorter? Such as `azure_monitor_alerts_for_job_failures` --> `alerts_for_job_failures` ?